pages/index.tsx

```diff
@@ -12,27 +12,30 @@ import * as Types from "../src/components/time-entries/TimeEntries.types";
 
 interface HomepageProps {
   timeEntries: Types.TimeEntryProps[];
+  clients: Types.ClientsProps[];
 }
 
 export const getServerSideProps = async () => {
   const timeEntries = await getTimeEntries();
+  const clients = await getClients();
 
   return {
     props: {
       timeEntries,
+      clients,
     },
   };
 };
 
-const Homepage = ({ timeEntries }: HomepageProps) => {
+const Homepage = ({ clients, timeEntries }: HomepageProps) => {
   return (
     <>
       <StoreProvider>
         <GlobalStyle />
         <ThemeProvider {...{ theme }}>
           <Header />
           <PageContainer>
-            <TimeEntries {...{ timeEntries }} />
+            <TimeEntries {...{ clients, timeEntries }} />
```

Comment:
It is usually only recommended to deconstruct props this way when you pass them to your Styled Components. So if it was `<Styled.Container {...{ isActive }} />` for example.